app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/mostPopularPatternsControls.jsx

```diff
@@ -144,12 +145,14 @@ export class MostPopularPatternsControls extends Component {
               intl.formatMessage(messages.attributeKeyValidationError),
             )}
           >
-            <AsyncAutocomplete
-              getURI={URLS.launchAttributeKeysSearch(activeProject)}
-              minLength={1}
-              creatable
-              placeholder={intl.formatMessage(messages.attributeKeyFieldPlaceholder)}
-            />
+            <FieldErrorHint error={intl.formatMessage(messages.attributeKeyValidationError)}>
```

Comment:
I suppose that `error` need to be removed here as it will be passed from `FieldProvider`.